src/resources/2023/GuidetoBurningLeaves.ts

```diff
@@ -0,0 +1,51 @@
+import {
+  Item,
+  Monster,
+  itemAmount,
+  visitUrl,
+} from "kolmafia";
+import { haveInCampground } from "../../lib";
+import { $item, $monster } from "../../template-string";
+
+
+const item = $item`A Guide to Burning Leaves`;
+
+export const specialLeaves: Map<Number, Item> = new Map([
```

Comment:
You can use `Map<number, Item | Monster>` if you want to include the monsters in this map.

Reply:
That being said, I would maybe use an object given that our keys are numbers.

Also worth noting: we never want to use the capitalized version of primitive types; only `number`, `boolean`, `string`, never `Number`, `Boolean` or `String`

Reply:
So, I'll submit what happens when I try this (made an object originally), it makes vscode unhappy.